app/pages/patients/patients.js

```diff
@@ -113,7 +113,7 @@ export let Patients = translate()(class extends React.Component {
         <div>
           {t('Tidepool provides free, secure data storage for diabetes data.')}
           <br />
-          {t('Would you like to set up data storage for someone’s diabetes data?')}
+          {t('Would you like to set up data storage for you or someone else\'s diabetes data?')}
```

Comment:
I think this needs to be "your" to be grammatically sound:
```suggestion
          {t('Would you like to set up data storage for your or someone else\'s diabetes data?')}
```

Reply:
Updated to `Would you like to set up data storage for yourself or for someone else's diabetes data?` as per product decision